src/scripts/mymodules/renderProjects.js

```diff
@@ -0,0 +1,193 @@
+import removeProject from './removeProject';
+// eslint-disable-next-line import/no-cycle
+import saveTodo from './saveTodo';
+// eslint-disable-next-line import/no-cycle
+import removeTodo from './removeTodo';
+
+const generatePrFieldDOM = (inputTitle, inputType, inputName, formClassName, todo) => {
+  const prField = document.createElement('div');
+  prField.classList.add('project__field');
```

Comment:
[Optional] You are repeating create elements and add class to it.
In this kind of case, you can create your own function that does the same thing and use it wherever you want to do it so that you can make your code shorter and simpler.

Reply:
#  Status: require Changes ♻️ 

Great job, @@@,  👍 💯 
You have made impressive work👏, but you still have some minor changes to take care of.

Please check the comments below: 

- [ ] This pull request title is great, but the description is not enough to describe your whole work for this pull request. It would be added more description. You'll see what is the ideal description of the pull request [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
<img width="935" alt="Screen Shot 2021-02-22 at 11 49 09" src="https://user-images.githubusercontent.com/53027195/108651065-59180f00-7504-11eb-9649-69d8a61bbc06.png">

- [ ] Please add the "Suggested newsletters for you" section at the bottom of the page:
![Artboard](https://user-images.githubusercontent.com/53027195/108651666-a0eb6600-7505-11eb-9263-67c1eb92b91b.png)

-[ ] Please add this project title to the title section of your reame file
![Artboard](https://user-images.githubusercontent.com/53027195/108652021-7057fc00-7506-11eb-88fc-02512ecfd21e.png)

## Required Changes ✏️💡 :

Please make all the required changes before you submit another code review.

**Happy Coding 😄**